ehsm_kms_service/function.js

```diff
@@ -325,8 +304,9 @@ const params_sort_str = (sign_params) => {
     }
     return str
   } catch (error) {
-    res.send(_result(404, 'Not Found', {}))
+    console.info('params_sort_str :: ', error)

```

Comment:
what does it mean "params_sort_str"?

Reply:
the params_sort_str is funcion name, it will be remove and change to user logger.